exp/gpu-support.md

```diff
@@ -1,40 +1,45 @@
-Anbox Cloud has support to manage GPUs and can provide them to individual containers for rendering and video encoding functionality.
+Anbox Cloud has support for managing GPUs and can provide them to individual containers for rendering and video encoding functionality.
 
-In case that no GPU is available Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
+If no GPU is available, Anbox Cloud automatically falls back to software rendering and video encoding. This makes it possible to run entirely without a GPU.
 
+<a name="supported-gpus"></a>
 ## Supported GPUs
 
 Anbox Clouds allows access to GPUs from Intel, AMD and Nvidia inside the Anbox container. Concrete support for the individual GPU depends on the platform being using for Anbox. The included `webrtc` platform currently supports the following GPUs:
 
-| Vendor | Model          | Render | Hardware Video Encode |
+| Vendor | Model          | Render | Hardware video encode |
 |--------|----------------|--------|-----------------------|
 | AMD    | WX5100, WX4100 | Yes    | No                    |
 | Nvidia | Quadro, Tesla  | Yes    | Yes                   |
```

Comment:
While we're on it we need to clean this up. For NVIDIA we should say `Quadro, Tesla, Ampere`